contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,163 @@
+
+> **Note:** As a result of putting persons on events we now have to be strict about identified and unidentified users. Now, only anonymous users can be aliased into other users - meaning that once you call `identify` on a user, you can no longer alias the user into another identified user. Because of that, we recommend double checking your `identify` and `alias` calls, and following advice above [how to upgrade a self-hosted instance to 1.41.0](/blog/the-posthog-array-1-41-0). 
+
+### New: Count of events per user
+
+![count events per user](../images/blog/array/1-41-0-count-events-per-user.png)
+
+Have you ever asked yourself '*What's the average average number of purchases per user?*' or '*What's the maximum number of forms submitted per user?'* 
+
+Questions like these used to be hard to answer with PostHog, but no more! We've added a new series option for insights, enabling you to see the count of events per user - including sub-options for seeing the averages, minimum and maximum count of events per user. 
+
+**Bonus update:** While working on this feature, we also took the opportunity to improve our math selector UI.
+
+### New: Text cards on dashboards
+
+![text cards on dashboards](../images/blog/array/1-41-0-text-card-on-dashboard.png)
+
+Previously there was no easy way to add context or links to a dashboard, meaning you may have to send long explanations when sharing a dashboard with teammates. That's why we've added the option for users on paid plans to add text cards where they can add any information they want, including metadata, images or gifs!
+
+### New: Ingestion warnings
+
+![ingestion warnings](../images/blog/array/1-41-0-ingestion.png)
+
+We've added a new page to the Data Management section which lists warnings related to data ingestion from the past 30 days. If you still try to merge identified users into others, the Ingestion Warning page is where we'll remind you that the merge got blocked.
+
+> **Note:** Self-hosted users managing kafka separately should create a new topic `clickhouse_ingestion_warnings` manually.
+
+
+### New: App metrics
+
+![app metrics](../images/blog/array/1-41-0-app-metrics-insight.png)
+
+Curious how well your apps are doing? Previously, you may have had to pour over the AWS logs, but now you can head to the new app metrics page to find out how many events an app has processed, how many retries were attempted and what errors may have occured. Very handy. 
+
+Want to take a look? Head to the apps page in your instance and click the chart symbol for any installed app.  
+
+> **Note:** Self-hosted users managing kafka separately should create a new topic `clickhouse_app_metrics` manually.
```

Comment:
@joethreepwood A note on what some self-hosted users may need to do for this to work. Feel free to update this as you'd like.